content/docs/storage/dataset.md

```diff
@@ -61,26 +61,26 @@ To add data to the default dataset, you can use the example below, however using
 
 ```js
 // Import the Apify SDK into your project
-const Apify = require('apify');
+import { Actor } from 'apify';
 
-// The optional Apify.main() function performs the
-// actor's job and terminates the process when it is finished
-Apify.main(async () => {
-    // Add one item to the default dataset
-    await Apify.pushData({ foo: 'bar' });
+// ...
+// Add one item to the default dataset
+await Actor.pushData({ foo: 'bar' });
 
-    // Add multiple items to the default dataset
-    await Apify.pushData([{ foo: 'hotel' }, { foo: 'cafe' }]);
-});
+// Add multiple items to the default dataset
+await Actor.pushData([{ foo: 'hotel' }, { foo: 'cafe' }]);
```

Comment:
Missing `init` and `exit` ? I would probably include it everywhere to make sure people don't forget it as it might cause weird bugs.

Reply:
I did originally, but it kinda increases the size of some snippets quite a lot - was not sure whether we want to do it even for some limited examples or not. Also - there were these comments:

// The optional Apify.main() function performs the
// actor's job and terminates the process when it is finished

Do we want to use Actor methods or Crawlee storages directly here? If it's gonna be `Actor` nearly everywhere - then well - maybe it's a good idea to keep init/exit everywhere

Reply:
I would do it like this. If the example has an `import` statement, then it should have `init` and `exit` to make it runnable. If it's just some one liner or two liner showcasing a function, it does not need to have those. What do you think?

Reply:
And yeah, I would use `Actor.` methods everywhere unless there's an absolute need to import Crawlee, because there's some crawling in the example.

Reply:
Went through the changed places - kinda decided to add it everywhere - I guess it will not hurt to form this pattern - if you're using actor - use these two before/after your main code.